test/extension/install_extension.test_slow

```diff
@@ -110,6 +110,19 @@ SELECT id, first_name, last_name, email FROM PARQUET_SCAN('https://raw.githubuse
 9	Jose	Foster	jfoster8@yelp.com
 10	Emily	Stewart	estewart9@opensource.org
 
+statement ok
+INSTALL 'json';
```

Comment:
All of the code in this file is skipped after line 14, could you perhaps remove all of the code after the mode skip to prevent people from adding unreachable tests here in the future? We have a different mechanism of testing loading of extensions that runs the regular tests of each extension in the CI.